gramps/gen/lib/surname.py

```diff
@@ -69,6 +69,25 @@ def __init__(self, source=None, data=None):
         if data:
             self.unserialize(data)
 
+    def get_object_state(self):
```

Comment:
Since `Surname` inherits from `SecondaryObject` which in turn inherits from `BaseObject`, why do we need to define the `get_object_state` and `set_object_state` methods here?